### PR TITLE
Add saving and loading with localstorage

### PR DESCRIPTION
**Pull request by adammetal**

### CODE REVIEW

**Comment by adammetal** [comment]
Check this pls!

File: rss.js [hunk 1]
```diff
@@ -34,6 +32,7 @@ function News(news) {
 
     const itemDiv = Item({ title, desc, link }, () => {
       state.saved.push({ title, desc, link });
+      localStorage.setItem("saved-news", JSON.stringify(state.saved));
```
Comment (10Cdegree) [inline rss.js]: Save news is a magic constant, please move it to a variable.

Comment (10Cdegree) [reply rss.js]: Create a save and load functions.

File: rss.js [hunk 2]
```diff
@@ -78,7 +77,7 @@ function Item(item, onSave) {
 
 const state = {
   news: [],
-  saved: [],
+  saved: JSON.parse(localStorage.getItem("saved-news")) ?? [],
```
Comment (10Cdegree) [inline rss.js]: According to the previous comments, create a load function for saved news.

**Review by 10Cdegree — CHANGES_REQUESTED** [review]

**Review by 10Cdegree — APPROVED** [review]

**Pull request merged**